docs/notebooks/distributed/custom_loop_pmap_example.md

```diff
@@ -0,0 +1,335 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.1
+kernelspec:
+  display_name: Python 3
+  name: python3
+---
+
+```{code-cell}
+:id: SDYtl1XHEEjK
+
+# Copyright 2022 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+```
+
++++ {"id": "nCt2V9cCR-CF"}
+
+[![Open in Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/google/jaxopt/blob/main/docs/notebooks/distributed/custom_loop_pmap_example.ipynb)
+
++++ {"id": "N3RErasEJpMo"}
+
+`jax.pmap` example using JAXopt.
+=========================================
+The purpose of this example is to illustrate how JAXopt solvers can be easily
+used for distributed training thanks to `jax.pmap`. In this case, we begin by
+implementing data parallel training of a multi-class logistic regression model
+on synthetic data. General aspects to pay attention to include:
+* How to use `jax.lax` reduction operators such as `jax.lax.pmean` or
+ `jax.lax.psum` in JAXopt solvers by using custom `value_and_grad` functions.
+* How `jax.pmap` can be used to transform the solver's `update` method to easily
+  write custom data-parallel training loops.
+
+Running the notebook on Google Colab:
```

Comment:
```suggestion
To obtain the best performance on Google Colab we recommend:
```
(since the example will likely still work on CPU/GPU)

Reply:
Done, thank you!